src/core/common/logging.hpp

```diff
@@ -89,6 +89,9 @@ extern "C" {
 #define _OT_REGION_CLI_PREFIX "-CLI-----: "
 #define _OT_REGION_CORE_PREFIX "-CORE----: "
 #define _OT_REGION_UTIL_PREFIX "-UTIL----: "
+#if OPENTHREAD_CONFIG_THREAD_VERSION >= OT_THREAD_VERSION_1_2
```

Comment:
nit: how about we avoid feature-flagging these defines (here and below)?

Reply:
removed.